dev/com.ibm.ws.jmx.connector.client.rest/test/com/ibm/ws/jmx/connector/converter/JSONConverterTest.java

```diff
@@ -2946,20 +2947,14 @@ public void testWriteThrowable() {
         StringBuilder regex = new StringBuilder();
         StringWriter stackTrace = new StringWriter();
```

Comment:
local variables stackTrace and tempOut can be removed as they are no longer used.